src/configmanager.cpp

```diff
@@ -186,13 +186,13 @@ bool ConfigManager::load()
 			string[IP] = getGlobalString(L, "ip", "127.0.0.1");
 		}
 
-		string[MAP_NAME] = getGlobalString(L, "mapName", "forgotten");
+		string[MAP_NAME] = getGlobalString(L, "mapName", "atlas");
 		string[MAP_AUTHOR] = getGlobalString(L, "mapAuthor", "Unknown");
 		string[HOUSE_RENT_PERIOD] = getGlobalString(L, "houseRentPeriod", "never");
 		string[MYSQL_HOST] = getEnv("MYSQL_HOST", getGlobalString(L, "mysqlHost", "127.0.0.1"));
-		string[MYSQL_USER] = getEnv("MYSQL_USER", getGlobalString(L, "mysqlUser", "forgottenserver"));
+		string[MYSQL_USER] = getEnv("MYSQL_USER", getGlobalString(L, "mysqlUser", "atlas"));
 		string[MYSQL_PASS] = getEnv("MYSQL_PASSWORD", getGlobalString(L, "mysqlPass", ""));
-		string[MYSQL_DB] = getEnv("MYSQL_DATABASE", getGlobalString(L, "mysqlDatabase", "forgottenserver"));
+		string[MYSQL_DB] = getEnv("MYSQL_DATABASE", getGlobalString(L, "mysqlDatabase", "atlas"));
 		string[MYSQL_SOCK] = getEnv("MYSQL_SOCK", getGlobalString(L, "mysqlSock", ""));
```

Comment:
To do separately: I don't think it ever makes sense to have defaults for these values

Reply:
I agree